src/tip-opengov.ts

```diff
@@ -107,12 +107,19 @@ async function signAndSendCallback(
   result: ISubmittableResult,
 ): Promise<TipResult> {
   return await new Promise((resolve, reject) => {
+    const resolveSuccess = () => {
+      unsubscribe();
+      resolve({ success: true, tipUrl: getTipUrl(contributor.network) });
+    };
```

Comment:
Shouldn't reject also lead to `unsubscribe()` call?

Reply:
Right, good catch. Created separate PR: https://github.com/paritytech/substrate-tip-bot/pull/92